guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -162,6 +162,17 @@ If you copy to a different folder, update the `backup_dir` variable in the `/etc
 You can either mount the shared storage or copy the backup files to the `/backup/` folder on the target server.
 . Power off the source server.
 . Enter the following commands to register to the Customer Portal, attach subscriptions, and enable only the required subscriptions:
+** If the target server runs {EL}{nbsp}9:
```

Comment:
According to RHSSG, lead-in sentences are supposed to be complete sentences. You could use just the operating system  `{EL}{nbsp}9:`.

Reply:
I'm not convinced that "EL 8/9" would be clear enough. Instead I turned lines 165 and 175 into complete sentences.